packages/components/filters/filters-list/src/filters-list.tsx

```diff
@@ -1,11 +1,135 @@
+import { type ReactNode, useState } from 'react';
+import { css } from '@emotion/react';
+import { MenuProps, MenuListProps } from 'react-select';
+import DropdownMenu from '@commercetools-uikit/dropdown-menu';
+import FilterMenu from '@commercetools-uikit/filter-menu';
+import SecondaryButton from '@commercetools-uikit/secondary-button';
+import SelectInput from '@commercetools-uikit/select-input';
+
+export type TAppliedFilterValue = {
+  filterKey: string;
+  /** does ReactNode | ReactNode[] make sense for single/multiple labels (shown in FilterMenu Chips) */
+  label: ReactNode[];
+};
+
+export type TFiltersConfiguration = {
+  key: string;
+  groupKey?: string;
+  label: ReactNode;
+  filter: ReactNode;
+  isPersistent?: boolean;
+  isDisabled?: boolean;
+  onClearRequest: Function;
+  onApplyRequest?: Function;
+  onSortRequest?: Function;
+};
+
+export type TFilterGroup = {
+  key: string;
+  label: ReactNode;
+};
+
 export type TFiltersListProps = {
-  /**
-   * This is a stub prop
-   */
-  label?: string;
+  filters: TFiltersConfiguration[];
+  filterGroups?: TFilterGroup[];
+  appliedFilters: TAppliedFilterValue[];
+  onClearAllRequest: Function;
+  onAddFilterRequest?: Function;
 };
 
+const CustomSelectMenu = ({
+  children,
+  innerProps: { ref, ...restInnerProps },
+}: MenuProps) => (
+  <div ref={ref} {...restInnerProps}>
+    {children}
+  </div>
+);
+
+const CustomMenuList = ({
+  children,
+  innerProps: { ref, ...restInnerProps },
+}: MenuListProps) => (
+  <div ref={ref} {...restInnerProps}>
+    {children}
+  </div>
+);
+
 function FiltersList(props: TFiltersListProps) {
-  return <div>{props.label}</div>;
+  const [selectedFilters, setSelectedFilters] = useState<string[]>([]);
+
+  const persistedFilters = props.filters
```

Comment:
I'm going to be the first to admit that there is work to be done on the internals of how we define persistent/selected/active filters (that show up in the filter bar)

Reply:
Out of curiosity: What's the definition of a `persistent` filter? And does the component need to know what a persistent filter is? (Or is it enough if it knows that it needs to be rendered or not (e.g. via show/hide prop on a filter-config item).

Reply:
According to [the figma](https://www.figma.com/design/b2S3GGpScVDuf15VAGEqeL/Filter-pattern?node-id=2339-6256&m=dev),

> Persistant filter-tag is sort of a quickfilter. It is displayed per default and can’t be removed by the “x” button

I don't know if a show/hide is enough, because the point of a "persistent" filter is that it's not removable from the list, which means that the `FiltersList` needs to know that in order to always display it, and the `FilterMenu` needs to know it to not display an `x`/remove button.

Reply:
The date filter in Audit Log is an example of a potential persistent filter, just in case you are curious.